hybrid/dev/kubernetes/agent.yaml

```diff
@@ -0,0 +1,263 @@
+---
+# Source: spire-agent/templates/serviceaccount.yaml
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: spire-agent
+  namespace: spire
+---
+# Source: spire-agent/templates/configmaps.yaml
+# ConfigMap containing the SPIRE agent configuration.
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: spire-agent
+  namespace: spire
+data:
+  agent.conf: |
+    agent {
+      data_dir = "/run/spire"
+      log_level = "DEBUG"
+      server_address = "spire-server"
+      server_port = "8081"
+      socket_path = "/run/secrets/workload-spiffe-uds/socket"
+      trust_bundle_path = "/run/spire/bundle/root-cert.pem"
+      trust_domain = "cluster.local"
+    
+    }
+
+    plugins {
+       NodeAttestor "hybrid" {
+        plugin_cmd = "/usr/local/bin/agentattestor"
+        plugin_data {
+          plugins {
+            k8s_psat {
+              cluster = "hybrid-node-attestor"
```

Comment:
Can we have use an environment variable to set this value since it is required to be in sync with the integration tests?

Reply:
Done